src/provider/hyperkzg.rs

```diff
@@ -185,7 +185,7 @@ where
   type Commitment = Commitment<E>;
   type CommitmentKey = CommitmentKey<E>;
 
-  fn setup(_label: &'static [u8], n: usize) -> Self::CommitmentKey {
+  fn setup(_label: &'static [u8], _blinding_label: &'static [u8], n: usize) -> Self::CommitmentKey {
```

Comment:
Nit: Do we need a separate label for blind generation?

Reply:
No, we don't. I was trying to create the h generator with a separate call to `from_label` - no reason to do that. I'll change it.